Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/Introduction.js

```diff
@@ -1,37 +1,44 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
-import { Editor } from '@tinymce/tinymce-react';
+import SunEditor from "suneditor-react";
+import 'suneditor/dist/css/suneditor.min.css';
+import htmlToText from 'html-to-text';
 
 export default function Introduction() {
-  
-  let handleEditorChange = (content, editor) => {
+
+  let handleChange = (content) => {
     //TODO Implement
-    console.log('HTML String:', content);
-    console.log('Plain Text:', editor.getContent({format: 'text'}));
-  }
+    console.log(content);
+    console.log(htmlToText.fromString(content));
+  };
+
   return (
     <div>
       <Typography variant="h4" >
         Introduction:
       </Typography>
-      <Editor
-        apiKey="sd525gpjpg57p2irp3p3nsdjmxj0j708hdfxp3iu0fdw3455"
-        initialValue="<p>Write in your introduction for your component.</p>"
-        init={{
-          height: 350,
-          menubar: true,
-          plugins: [
-            'advlist autolink lists link image charmap print preview anchor',
-            'searchreplace visualblocks code fullscreen',
-            'insertdatetime media table paste code help wordcount'
-          ],
-          toolbar:
-            'undo redo | formatselect | bold italic backcolor | \
-            alignleft aligncenter alignright alignjustify | \
-            bullist numlist outdent indent | removeformat | help'
+      <SunEditor 
+        setOptions={{
+          width:'100%',
+          height: 400,
+          placeholder: "Enter in introduction of component...",
+          buttonList: [
+            ['font', 'fontSize', 'formatBlock'],
```

Comment:
Similarly here, maybe check out some examples of how other sites handle this on mobile. Maybe only include the most important few options on mobile? Right now there is a ton of screen real estate being taken up by the buttons on mobile

Reply:
Yea I definitely change the options for mobile devices depending on screen width.

Reply:
Actually there is a way to do it within the component :)